claim/test_services.py

```diff
@@ -115,6 +115,8 @@ def test_extended_claim_submit_xml(self):
 
     @mock.patch('django.db.connections')
     def test_claim_submit_error(self, mock_connections):
+        if not connection.vendor == 'mssql':
```

Comment:
I think 'mssql' could be handled by const variable too.

Reply:
I will leave it like this. This connection.vendor should not be used anymore. Each new test should be compatibile with pssql and mssql so I will leave it like this